docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,616 @@ license: |
++------------------------------------+
+
+SELECT PERCENTILE(c1, ARRAY(0.25, 0.75), 10) FROM buildin_agg;
++-------------------------------------+
+|percentile(c1, array(0.25, 0.75), 10)|
++-------------------------------------+
+|                           [1.0, 4.0]|
++-------------------------------------+
+
+--PERCENTILE_APPROX and APPROX_PERCENTILE
+SELECT PERCENTILE_APPROX(c1, 0.25, 100) FROM buildin_agg;
++------------------------------------------------+
+|percentile_approx(c1, CAST(0.25 AS DOUBLE), 100)|
++------------------------------------------------+
+|                                               1|
++------------------------------------------------+
+
+SELECT APPROX_PERCENTILE(c1, 0.25, 100) FROM buildin_agg;
++------------------------------------------------+
+|approx_percentile(c1, CAST(0.25 AS DOUBLE), 100)|
++------------------------------------------------+
+|                                               1|
++------------------------------------------------+
+
+SELECT PERCENTILE_APPROX(c1, ARRAY(0.25, 0.85), 100) FROM buildin_agg;
++---------------------------------------------+
+|percentile_approx(c1, array(0.25, 0.85), 100)|
++---------------------------------------------+
+|                                       [1, 4]|
++---------------------------------------------+
+
+SELECT APPROX_PERCENTILE(c1, array(0.25, 0.85), 100) FROM buildin_agg;
++---------------------------------------------+
+|approx_percentile(c1, array(0.25, 0.85), 100)|
++---------------------------------------------+
+|                                       [1, 4]|
++---------------------------------------------+
+
+--SKEWNESS
+SELECT SKEWNESS(c1) FROM buildin_agg;
++----------------------------+
+|skewness(CAST(c1 AS DOUBLE))|
++----------------------------+
+|          0.5200705032248686|
++----------------------------+
+
+SELECT SKEWNESS(col) FROM VALUES (-1000), (-100), (10), (20) AS TAB(col);
++-----------------------------+
+|skewness(CAST(col AS DOUBLE))|
++-----------------------------+
+|          -1.1135657469022011|
++-----------------------------+
+
+--STDDEV_SAMP, STDDEV and STD
+SELECT STDDEV_SAMP(c1) FROM buildin_agg;
++-------------------------------+
+|stddev_samp(CAST(c1 AS DOUBLE))|
++-------------------------------+
+|              1.618347187425374|
++-------------------------------+
+
+SELECT STDDEV(c1) FROM buildin_agg;
++--------------------------+
+|stddev(CAST(c1 AS DOUBLE))|
++--------------------------+
+|         1.618347187425374|
++--------------------------+
+
+SELECT STD(c1) FROM buildin_agg;
++-----------------------+
+|std(CAST(c1 AS DOUBLE))|
++-----------------------+
+|      1.618347187425374|
++-----------------------+
+
+--STDDEV_POP
+SELECT STDDEV_POP(c1) FROM buildin_agg;
++------------------------------+
+|stddev_pop(CAST(c1 AS DOUBLE))|
++------------------------------+
+|             1.498298354528788|
++------------------------------+
+
+--SUM
+SELECT SUM(col) FROM VALUES (5), (10), (15) AS TAB(col);
```

Comment:
nit: `TAB` -> `t` (I think lowercases are better except for the SQL keywords.)

Reply:
sure